### PR TITLE
Auth48

### PR DESCRIPTION
**Pull request by martinthomson**
These are all the changes I found from the text diff, plus some extra changes about versions.

### CODE REVIEW

**Review by DavidSchinazi — APPROVED** [review]

File: draft-ietf-quic-bit-grease.md [hunk 1]
```diff
@@ -36,17 +36,17 @@ arbitrary value for the second-most significant bit in QUIC packets.
 
 # Introduction
 
-QUIC {{!QUIC=RFC9000}} intentionally describes a very narrow set of fields that
-are visible to entities other than endpoints.  Beyond those characteristics that
-are defined as invariant {{?QUIC-INVARIANTS=RFC8999}}, very little about the
-"wire image" {{?RFC8546}} of QUIC is visible.
+QUIC Invariants {{!QUIC-INV=RFC8999}} intentionally describes a very narrow set
```
Comment (LPardue) [inline draft-ietf-quic-bit-grease.md]: I know you probably used `QUIC-INV` on purpose but it makes it less consistent with a how we reference it in RFC 9000. It's editorial, you decide, just giving some 2c

**Review by LPardue — APPROVED** [review]
LGTM modulo one tiny nit

File: draft-ietf-quic-bit-grease.md (hunk 1 above)
Comment (martinthomson) [reply draft-ietf-quic-bit-grease.md]: Yeah, I took another go at it in a later commit after realizing that.

Comment (LPardue) [reply draft-ietf-quic-bit-grease.md]: Oh deep I looked only at 1 commit

**Review by huitema — APPROVED** [review]
I have reviewed the changes, and I agree that they are editorial in nature.

**Review by mirjak — APPROVED** [review]

**Pull request merged**